src/Optimize/tests/test_variable_set.cpp

```diff
@@ -35,7 +36,11 @@ TEST_CASE("VariableSet empty", "[optimize]")
 TEST_CASE("VariableSet one", "[optimize]")
 {
   VariableSet vs;
-  double first_val = 1.123456789;
+  #ifdef QMC_COMPLEX
+  std::complex<double> first_val(1.123456789, 0.0);
+  #else
+  double first_val=1.123456789;
+  #endif
   vs.insert("first", first_val);
```

Comment:
ValueType first_val(1.123456789);
should work

Reply:
Fixed